### ISSUE TITLE
COVID-19 RDS Help

### ISSUE THREAD

**Issue by tagquijano** [AlanSayler/RDS#2]
Hi, I hope you are well during this hectic time. I'm working on a RDS for vulnerable populations affected by COVID and I found your code. Currently going through it trying to understand (I'm somewhat new to python but am excited to use it). Wondering if I could see exactly how the questions are structured on the survey end itself. That was the one piece I think I am missing because I can't see how exactly I need to structure my questions to work with the code.

Thanks,
Tag Quijano

**Comment by MVRick90** [comment]
I am also going to be using Qualtrics to do RDS. @tagquijano or @AlanSayler, Would you be willing to have a brief call to discuss your work and how you used Qualtrics?

**Comment by tagquijano** [comment]
Hi, hope all is well. I mostly just modified Alan’s original RDS system with his help, but if he is not available for a call I’d be happy to try to help out. I’m not an expert in python but am familiar with the system. Let me know your availability.

--
Thomas "Tag" Quijano
Yale College BA '21
Yale School of Public Health MPH '22
1-609-712-3599
tagquijano@gmail.com

**Comment by MVRick90** [comment]
Thomas,

Thank you so much for being willing to help.

The Github documentation was not very clear. Could you outline how python
integrated with Qualtrics?

I'm currently in the middle of setting up an RDS Qualtrics recruitment
protocol and wanted to see if others had experience and would be willing to
share what they learned. Would you be open to sharing how your Qualtrics
system works?

Depending on your level of contribution, our PI might consider providing
credit in publications related to this study.

My schedule is flexible next week. What's your availability? I can send out
a meeting invite for a quick 30 minute call.

-Rick
Dartmouth College BA '12
SDSU MS'21

Center on Gender Equity and Health

Division of Global Public Health

University of California, San Diego

9500 Gilman Drive

La Jolla, CA 92093-0507

rveramonroy@health.ucsd.edu

619-277-8303

http://geh.ucsd.edu/
<https://urldefense.com/v3/__http:/geh.ucsd.edu/__;!!LLK065n_VXAQ!z3M1wFhewBT6XKU5Q0ycWegQhnGnKuJU7grumqjN2rdBjFVcRFfbmmFdf7jkVqcGYgg7$>

@GEH_UCSD

**Comment by tagquijano** [comment]
Do you have any time Friday between 2 and 4 EST?

--
Thomas "Tag" Quijano
Yale College BA '21
Yale School of Public Health MPH '22
1-609-712-3599
tagquijano@gmail.com

**Comment by MVRick90** [comment]
Yes, I'm available anytime 2-4EST (11AM-1PM PST). I've sent you a Zoom
invite for 11AM PST on March 12 to your email listed in your email
signature.

Here is the link just in case:
https://uchealth.zoom.us/j/85080521690

Thank you,
-Rick

**Comment by tagquijano** [comment]
Sounds good. Looking forward to it.

--
Thomas "Tag" Quijano
Yale College BA '21
Yale School of Public Health MPH '22
1-609-712-3599
tagquijano@gmail.com
